lib/js/components/Banner/Banner.vue

```diff
@@ -333,7 +333,7 @@ export default defineComponent({
 		color: {
 			type: String,
 			default: BANNER_COLORS.DEFAULT,
-			validator: (color) => Object.values(BANNER_COLORS).includes(color),
+			validator: (color) => Object.values(BANNER_COLORS).includes(color as BannerColor),
```

Comment:
Using `as` always makes me suspicious. It suggests that TS cannot handle types properly, or I did something wrong with types. It doesn't change much in this case, as we don't call `validator()` from our code, but it looks better for me if we move type definition into parameter definition
Similar change can be applied in other places

```suggestion
			validator: (color: BannerColor) => Object.values(BANNER_COLORS).includes(color),
```